src/ensembl/src/content/app/browser/browser-bar/BrowserBar.tsx

```diff
@@ -207,7 +208,9 @@ export const BrowserInfo = ({ ensObject }: BrowserInfoProps) => {
           <dd className={styles.ensObjectLabel}>
             <label>Region: </label>
             <span className={styles.value}>
-              {`${ensObject.location.chromosome}:${ensObject.location.start}:${ensObject.location.end}`}
+              {`${ensObject.location.chromosome}:${getCommaSeparatedNumber(
+                ensObject.location.start
+              )}:${getCommaSeparatedNumber(ensObject.location.end)}`}
```

Comment:
Would it be better to have a shared function that takes the `location` and returns the formatted string?

Reply:
As we discussed offline the `ensObject` shape is going to change, so I am thinking of waiting till that is finalised in order create this shared function.